json_schema/type/project/project.json

```
@@ -71,55 +71,55 @@
             "type": "array",
             "items": {
                 "type": "string",
-                "pattern": "^[D|E|S]RP[0-9]+$"
+                "pattern": "^PRJ[E|N|D][A-Z]\\d+$"
             },
-            "example": "SRP000000",
+            "example": "PRJNS000000",
             "user_friendly": "INSDC project accession",
-            "guidelines": "Enter accession if project has been archived. Accession can be from the DDBJ, NCBI, or EMBL-EBI and must start with DRP, SRP, or ERP, respectively."
+            "guidelines": "Enter accession if project has been archived. Accession can be from the DDBJ, NCBI, or EMBL-EBI and must start with PRJD, PRJN, or PRJE, respectively."
         },
         "geo_series_accessions": {
             "description": "A Gene Expression Omnibus (GEO) series accession.",
             "type": "array",
             "items": {
                 "type": "string",
-                "pattern": "^GSE.*$"
+                "pattern": "^GSE\\d+$"
             },
             "example": "GSE00000",
             "user_friendly": "GEO series accession",
-            "guidelines": "Enter accession if project has been archived. Accession must start with GSE."
+            "guidelines": "Enter accession if project has been archived in GEO. Accession must start with GSE."
         },
         "array_express_accessions": {
             "description": "An ArrayExpress accession.",
             "type": "array",
             "items": {
                 "type": "string",
-                "pattern": "^E-....-.*$"
+                "pattern": "^E-[A-Z]{4}-\\d+$"
             },
             "example": "E-AAAA-00",
             "user_friendly": "ArrayExpress accession",
-            "guidelines": "Enter accession if project has been archived. Accession must start with E-."
+            "guidelines": "Enter accession if project has been archived in ArrayExpress. Accession must start with E-."
         },
         "insdc_study_accessions": {
             "description": "An International Nucleotide Sequence Database Collaboration (INSDC) study accession.",
             "type": "array",
             "items": {
                 "type": "string",
-                "pattern": "^PRJ[E|N|D][a-zA-Z][0-9]+$"
+                "pattern": "^[D|E|S]RP\\d+$"
```

This regex matches `|RP123`. So either, 

```suggestion
                "pattern": "^[DES]RP\\d+$"
```

or 

```suggestion
                "pattern": "^(D|E|S)RP\\d+$"
```

But the former is far more common when the alternatives are individual characters. `|` is typically used when the alternatives are complex regexes in their own right.